src/energyplus/ForwardTranslator/ForwardTranslateSizingZone.cpp

```diff
@@ -243,7 +243,7 @@ namespace energyplus {
     // This would be done in forwardTranslateControllerMechanicalVentilation except doing it here maintains proper order of the idf file.
 
     // Now that Multiple AirLoopHVACs serving the same zone are possible, need to loop on all
-    // NOTE: translateControllerMechnicalVentilation ensures ControllerMechanicalVentilation doesn't end up with no extensible groups!
+    // NOTE: translateControllerMechanicalVentilation ensures ControllerMechanicalVentilation doesn't end up with no extensible groups!
```

Comment:
@kbenne I noticed this. The Controller:MechanicalVentilation extensible groups (the Zone/DSOA groups) are NOT filled if the Sizing:Zone object isn't translated, and it's been like this since the first github commit:

https://github.com/NREL/OpenStudio/blob/c3a44f5d935a274d1d833323933d7df7f53e21dc/openstudiocore/src/energyplus/ForwardTranslator/ForwardTranslateSizingZone.cpp#L260-L300